### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pmattam**
@leecmoses...please take a look at this
@aparker22...please take a look at this

### CODE REVIEW

**Comment by leecmoses** [comment]
👍🏻

**Comment by aparker22** [comment]
You definitely were able to make your code more succinct; I can read what is happening in each step.  There are some additional functions that you could replace with the fat arrow => if you wanted to.

**Comment by illiaChaban** [comment]
@leecmoses bad job, mo
@aparker22 good job, ashley !

**Comment by leecmoses** [comment]
@illiaChaban no food for you

File: phonebookServerRefactor.js [hunk 1]
```diff
@@ -58,31 +62,35 @@ let notFound = function(request, response) {
...
-//     { method: 'POST', path: '/contacts', handler: postContacts }
-// ];
-var getHandler = function(method, path) {
-    let handler = (method === 'POST' && path === '/contacts') ? postContacts : (method === 'PUT') ? updateContact :
-        (method === 'DELETE') ? deleteContact : (method === 'GET' && path === '/contacts') ?
-        getContacts : (method === 'GET' && path === '/contacts/') ? getContact : notFound;
-    return handler;
-};
+let routes = [
+    { method: 'GET', path: '/contacts/', handler: getContact },
```
Comment (itzik415) [inline phonebookServerRefactor.js:66]: This part is looking great!

**Comment by itzik415** [comment]
@pmattam good job!!

**Comment by aparker22** [comment]
@leecmoses Does that mean I get food?

**Comment by leecmoses** [comment]
@aparker22 Anytime 😁

**Review by itzik415 — APPROVED** [review]

**Pull request merged**